doc/concepts.rst

```diff
@@ -347,6 +347,20 @@ all the literals evaluate to true. It is an error if none of the clauses are
 selectable. You can add a fallback by adding a clause of the form ``(->
 <file>)`` at the end of the list.
 
+Re-exported dependencies
+------------------------
+
+A dependency ``foo`` may be marked as *re-exportable* using the following
+syntax:
+
+.. code:: scheme
+
+   (re_export foo)
+
+This states that this library explicitly re-exports the interface of ``foo``.
+This is primarily useful when :ref:`implicit transitive dependencies<implicit_transitive_deps>`
+are disallowed, so it's documented along with the general feature.
```

Comment:
I don't understand the last part. What should be documented along with the general feature?

Reply:
I just meant that the main documentation for how `re_export` works is in the implicit transitive deps section. Since it only makes sense to explain it the context of that feature.

Reply:
Ack. it seems a bit odd to me that the documentation has to justify itself. I tweak the text. I also changed "re-exportable" to "re-exported" as a dependency marked with `re_export` will systematically be re-exported.